TunnelVisionLabs.ReferenceAssemblyAnnotator/NullableAttributes.cs

```diff
@@ -84,4 +84,61 @@ internal sealed class DoesNotReturnIfAttribute : Attribute
         /// <summary>Gets the condition parameter value.</summary>
         public bool ParameterValue { get; }
     }
+
+    /// <summary>Specifies that the method or property will ensure that the listed field and property members have not-null values.</summary>
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Property, Inherited = false, AllowMultiple = true)]
+    internal sealed class MemberNotNullAttribute : Attribute
```

Comment:
📝 These need to be conditionally included for .NET Standard 2.1 and .NET Core 3.x builds when `GenerateNullableAttributes` is set. See here for an example:

https://github.com/dotnet/roslyn/blob/bc85a40ac042f9cf799f5e43deb0fee2d595247b/src/Compilers/Core/Portable/InternalUtilities/NullableAttributes.cs#L90-L94

This will require conditional directives in the source file, and modifications to the handling of `GenerateNullableAttributes` in the build targets.

Reply:
Hum yeah, great point. I'll give it a try when I have a moment.